### ISSUE TITLE
Multiple definition of delete_cmd() prevents compilation of IBM1130 sim in 15-hour old commit :)

### ISSUE THREAD

**Issue by tangentsoft** [simh/simh#404]
Commit [7a1b35ae12f] causes this complaint on macOS:

<pre>Ibm1130/ibm1130_prt.c:653:15: error: static declaration of 'delete_cmd' follows non-static
      declaration
static t_stat delete_cmd (int32 flag, CONST char *cptr)
              ^
./scp.h:93:8: note: previous declaration is here
t_stat delete_cmd (int32 flg, CONST char *cptr);
</pre>

It seems both SCP and the IBM1130 simulator have a function called `delete_cmd()` and the former is exported system-wide, colliding with the latter's private version.  I've just temporarily diked the latter out so I can get on with testing Mark's recent PDP-8 change.

Incidentally, I think this calls into question your Makefile's dependency rules, since it seems that touching scp.h didn't force a rebuild of ibm1130_prt.c there, else you wouldn't have checked the code in with this build error. The PiDP-8/I project's Makefile has a fairly spiffy autodependency generation system that's much harder to fool than hand-written rules. Feel free to [steal it](https://tangentsoft.com/pidp8i/file/Makefile.in). Our code is fairly heavily tuned from the original, linked in the Makefile comments.

**Comment by markpizz** [comment]
Thanks for finding this.  

Honestly, my primary development environment is Windows and the Microsoft compiler actually allows the local static function to exist with the globally declared one being defined as well.  However, since this change was simple and completely simulator independent I only tested a small simulator (small meaning quick to compile) on Linux and OS X before I committed this change.  I'll fix later.

The makefile would have built all simulators had I not merely typed "make sds"... :-)
